lib/SILGen/SILGenBridging.cpp

```diff
@@ -392,7 +392,8 @@ static void buildFuncToBlockInvokeBody(SILGenFunction &SGF,
   if (blockTy->getNumResults() != 0) {
     auto result = blockTy->getSingleResult();
     if (result.getConvention() == ResultConvention::Indirect) {
-      indirectResult = entry->createFunctionArgument(blockResultTy);
```

Comment:
It doesn't seem to make any difference, but I wonder whether `blockConv.getSingleSILResultType(SGF.getTypeExpansionContext()))` should be passed instead of `blockResultTy.getAddressType()` to `createFunctionArgument`.

Reply:
I think what you've got is right.